backend/file-system/src/main/java/com/htec/filesystem/service/FolderService.java

```diff
@@ -31,4 +49,19 @@ public boolean initializeFolders(Long id) {
         return new File(userShelvesPath).mkdirs();
     }
 
+    public ResponseEntity<List<FileDTO>> getFiles(Long id) {
```

Comment:
rename id to folderId

Reply:
try to avoid name of entry params in method to be 'id', try to specify some concrete name

Reply:
resolved